node-package/README.md

```diff
@@ -0,0 +1,38 @@
+# node-package Docker Image
+
+## Description
+
+node-package is a Docker image based on node:12 that comes with a pre-installed node_modules.zip based on the Data Commons dc/website/package.json.
```

Comment:
can you use the full url for package.json:

[package.json](https://github.com/datacommonsorg/website/blob/master/static/package.json)

Reply:
also worth mentioning package-lock.json since it's crucial here

Reply:
Fixed